pkg/shardddl/optimism/lock_test.go

```diff
@@ -1580,3 +1584,81 @@ func newInfoWithVersion(task, source, upSchema, upTable, downSchema, downTable s
 	info.Version = vers[source][upSchema][upTable]
 	return info
 }
+
+func (t *testLock) TestLockTrySyncReceived(c *C) {
```

Comment:
which case does it test?

Reply:
rename in 814d7a7 . Now we use table-info-before for all table, create a different index will not be in conflict now.